js/src/ui/ParityBackground/parityBackground.js

```diff
@@ -16,26 +16,17 @@
 
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
-import { bindActionCreators } from 'redux';
 
 class ParityBackground extends Component {
-  static contextTypes = {
-    muiTheme: PropTypes.object.isRequired
-  }
-
   static propTypes = {
+    style: PropTypes.object.isRequired,
```

Comment:
Couldn't it just have been done in `componentWillMount`?

I think the single calculation is great, just concerned about "special cases" where we start doing things inconsistently.